DESCRIPTION

```diff
@@ -23,7 +23,10 @@ Suggests:
   knitr,
   aws.signature,
   aws.s3,
-  testthat
+  testthat,
+  foreach,
+  doParallel,
+  parallel
```

Comment:
Noting that these are in the Suggests section. We probably need a `requireNamespace(x, quietly = TRUE)` call for each of them before they're used in `loop_tasks`, right? (http://r-pkgs.had.co.nz/description.html)